main.tex

```diff
@@ -343,28 +343,40 @@ \subsubsection{Normalization layers}
 
 Batch, instance, and layer normalization are often informally described using the same
 equation, but they each correspond to very different functions. They differ
-by which axes are normalized.
+both by which axes are \textit{standardized} as well as their parameters.
 
-We can define a single generic normalization layer:
+We can define a single generic standardization function as:
 \begin{align*}
-  \nfun{\ax}{XNorm} \colon \mathbb{R}^{\ax} &\rightarrow \mathbb{R}^{\ax} \\
-  \nfun{\ax}{XNorm}(X; \gamma, \beta, \epsilon) &= \frac{X - \nfun{\ax}{mean}(X)}{\sqrt{\nfun{\ax}{var}(X)} + \epsilon} \odot \gamma + \beta
+  \nfun{\ax}{standardize} \colon \mathbb{R}^{\ax} &\rightarrow \mathbb{R}^{\ax} \\
+  \nfun{\ax}{standardize}(X) &= \frac{X - \nfun{\ax}{mean}(X)}{\sqrt{\nfun{\ax}{var}(X) + \epsilon}}
```

Comment:
Technically this should depend on \epsilon, right?

Reply:
People seem to use a fixed constant. I noted that.